boards/ct/ctcc/Kconfig

```diff
@@ -3,7 +3,14 @@
 # Copyright (c) 2024 CTHINGS.CO
 # SPDX-License-Identifier: Apache-2.0
 
+if BOARD_CTCC
+
+if BOARD_CTCC_NRF52840
+
 config BOARD_SERIAL_BACKEND_CDC_ACM
 	bool "USB CDC"
 	default y
-	depends on BOARD_CTCC_NRF52840
+
+endif # BOARD_CTCC_NRF52840
+
+endif # BOARD_CTCC
```

Comment:
why?

Reply:
The why? comment was for why is this file being changed at all, and it still is?

Reply:
Yes, sorry, I misunderstood. It is probably some leftover, you're right, it shouldn't be changed. Reverted